Missing `docbook-xml` Debian packaging README

`packaging/debian/README.md` does not include installation of `docbook-xml` which provides missing local `/usr/share/xml/docbook/schema/dtd/4.5/docbookx.dtd` file.

```
make deb
[...]
a2x -D docs/man/man1/ -d manpage -f manpage docs/man/man1/ansible.1.asciidoc
a2x: WARNING: --destination-dir option is only applicable to HTML based outputs
a2x: ERROR: "xmllint" --nonet --noout --valid "<snip>/docs/man/man1/ansible.1.xml" returned non-zero exit status 4
Makefile:117: recipe for target 'docs/man/man1/ansible.1' failed
make: *** [docs/man/man1/ansible.1] Error 1
rm docs/man/man1/ansible.1.asciidoc
```

--- a/packaging/debian/README.md
+++ b/packaging/debian/README.md
@@ -4,7 +4,7 @@ Ansible Debian Package
 To create an Ansible DEB package:
 
     sudo apt-get install python-paramiko python-yaml python-jinja2 python-httplib2 python-setuptools python-six sshpass
-    sudo apt-get install cdbs debhelper dpkg-dev git-core reprepro python-support fakeroot asciidoc devscripts docbook-xml
+    sudo apt-get install cdbs debhelper dpkg-dev git-core reprepro python-support fakeroot asciidoc devscripts docbook-xml xsltproc
     git clone git://github.com/ansible/ansible.git
     cd ansible
     make deb

--- a/packaging/debian/README.md
+++ b/packaging/debian/README.md
@@ -4,7 +4,7 @@ Ansible Debian Package
 To create an Ansible DEB package:
 
     sudo apt-get install python-paramiko python-yaml python-jinja2 python-httplib2 python-setuptools python-six sshpass
-    sudo apt-get install cdbs debhelper dpkg-dev git-core reprepro python-support fakeroot asciidoc devscripts
+    sudo apt-get install cdbs debhelper dpkg-dev git-core reprepro python-support fakeroot asciidoc devscripts docbook-xml
     git clone git://github.com/ansible/ansible.git
     cd ansible
     make deb

